File: J1_08 - Empresa/empresa/src/Funcionario.java
public class Funcionario {
    private String nome;
    private double salario;
    private String dataDeAdmissao;

    public Funcionario(String nome, double salario, String dataDeAdmissão)
    {
        this.nome = nome;
        this.salario = salario;
        this.dataDeAdmissao = dataDeAdmissão;
    }

    public String getNome()
    {
        return this.nome;
    }

    public double getSalario()
    {
        return this.salario;
    }

    public void setSalario(double salario)
    {
        this.salario = salario;
    }
}

File: desc_CPP_02/Includes/Funcionario.cpp
#include "Funcionario.hpp"

Funcionario::Funcionario(const string nome, double salario, const string dataAdmissao) : nome(nome), salario(salario), dataAdmissao(dataAdmissao) {}

void Funcionario::darAumento(double percentual) {
    double aumento = salario * percentual;
    salario += aumento;
}

double Funcionario::getSalario() const { return salario; }

